config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-periodics.yaml

```diff
@@ -332,9 +332,5 @@ periodics:
         done;
         go work sync;
         cd -;
-        kubetest2 gce -v 2 --repo-root $REPO_ROOT \
-          --build --up --down --test=ginkgo \
-          --master-size n1-standard-2  \
```

Comment:
You are changing master and node size. Is it by design?

Reply:
I'm copying paste from the job just before this, to get a config that works

Reply:
We don't need bumped sizes of those machines. Please revert node/master size changes, let's save some CNCF's money.

Reply:
done